packages/kbn-handlebars/README.md

```diff
@@ -71,6 +84,8 @@ Some of the tests have been copied from the upstream `handlebars` project and mo
 
 If the script outputs a diff for a given file, it means that this file has been updated.
 
+_Note: that this will look for chanages in the `4.x` branch of the `handlebars.js` repo only. Changes in the `master` branch are ignored._
```

Comment:
__typo:__
```suggestion
_Note: that this will look for changes in the `4.x` branch of the `handlebars.js` repo only. Changes in the `master` branch are ignored._
```